edb/edgeql/parser/grammar/ddl.py

```diff
@@ -424,12 +424,22 @@ def reduce_AlterFinal(self, *kids):
 # CREATE DATABASE
 #
 class CreateDatabaseStmt(Nonterm):
-    def reduce_CREATE_DATABASE_AnyNodeName_OptCreateRoleCommandsBlock(
-        self,
-        *kids,
-    ):
+    def reduce_CREATE_DATABASE_regular(self, *kids):
+        """%reduce CREATE DATABASE AnyNodeName OptCreateDatabaseCommandsBlock
```

Comment:
And this commit actually adds a new feature to EdgeQL that needs to be documented and perhaps deserves a commit message.

Reply:
It's a hidden feature, only for `testmode`.  There is no intention to expose it for now.

Reply:
Ah, I see you raise an error in `_cmd_tree_from_ast`. All's good.